### ISSUE TITLE
Data ingestion fails at times, when there there is a lot of writes

### ISSUE THREAD

**Issue by aaruna** [tuplejump/stargate-core#69]
The client sees errors similar to

```
com.datastax.driver.core.exceptions.WriteTimeoutException: Cassandra timeout during write query at consistency ONE (1 replica were required but only 0 acknowledged the write)
```

The cassandra node has this error

```
ERROR [Thread-3] 2016-01-29 01:13:14,678 CassandraDaemon.java:227 - Exception in thread Thread[Thread-3,5,main]
java.lang.NullPointerException: null
	at com.tuplejump.stargate.RowIndex$1.run(RowIndex.java:159) ~[stargate-core-1.1.0.jar:na]
```

**Comment by aaruna** [comment]
Upon further investigation, noticed that ExceptionHandler was missing for the Disruptor.
Upon adding that, I have seen that there is a NullPointerException in IndexEventSubscriber:L45
```
final RowIndexSupport rowIndexSupport = indexingService.support.get(columnFamily.metadata().cfName);
```

Noticed that the consumers are consuming events before they are published.

**Comment by aaruna** [comment]
There is a bug in Disruptor 3.0.1 (where consumers could get ahead of publishers) which was fixed in 3.1.1
